docs/src/examples/custom_relative_factors.md

```diff
@@ -47,7 +58,7 @@ function getSample(cf::CalcFactor{<:Pose2Pose2})
 end
 ```
 
-The return type for `getSample` is unrestricted, and will be passed to the residual function "as-is".
+The return type for `getSample` is unrestricted, and will be passed to the residual function "as-is", but must return values representing a tangent vector for `<:AbstractRelative`
```

Comment:
I think this might be the other way around, `getSample` on `<:AbstractPrior` must return a Point on manifold, on Relative, it is unrestricted, but we recommend a tangent vector `TpM`. 
I'll try and find the issue where this was discussed.

Reply:
https://github.com/JuliaRobotics/IncrementalInference.jl/issues/1371

Reply:
https://github.com/JuliaRobotics/RoME.jl/issues/465

Reply:
Tangent on relative factors -- that's what the PR already has, no?  "a tangent vector for `<:AbstractRelative`"